assets/init/postgres-init

```diff
@@ -13,16 +13,21 @@ export C035="\e[35m"
 export C036="\e[36m"
 export C037="\e[37m"
 
-PGVERSION=$(postgres --version | egrep -o "[0-9]{1,}\.[0-9]{1,}")
+PGVERSION=$(postgres --version | egrep -o "1[0-9]")
```

Comment:
Why is the old regex wrong? What is this supposed to match?

It looks quite different to me:
```
[root@f7c7ec457e7a /]# postgres --version | egrep -o "1[0-9]"
12
11
[root@f7c7ec457e7a /]# postgres --version | egrep -o "[0-9]{1,}\.[0-9]{1,}"
12.11
```

Reply:
The upgrade command is based on the major version of postgresql, not the minor version anymore. The minor version was relevant for 9.x I think.